ui/src/app/subnets/views/SubnetsList/SubnetsTable/components.tsx

```diff
@@ -0,0 +1,191 @@
+import { memo, useState } from "react";
+
+import {
+  TableRow as TableRowComponent,
+  TableHeader as TableHeaderComponent,
+  TableCell as TableCellComponent,
+  Button,
+} from "@canonical/react-components";
+import type {
+  TableHeaderProps,
+  TableCellProps,
+} from "@canonical/react-components";
+
+import type { SubnetsTableRow, SubnetsTableColumn } from "./types";
+import { getRowPropsAreEqual } from "./utils";
+
+export const TableHeader = ({
+  label,
+  ...props
+}: TableHeaderProps & {
+  label: string;
+}): JSX.Element => (
+  <TableHeaderComponent {...props}>{label}</TableHeaderComponent>
+);
+
+const TableCell = ({
+  cellData,
+  children,
+  className,
+  ...props
+}: TableCellProps & {
+  cellData: SubnetsTableColumn;
+}): JSX.Element => (
+  <TableCellComponent
+    className={`${className} ${
+      cellData.isVisuallyHidden ? "u-no-border--top" : ""
+    }`}
+    {...props}
+  >
+    <span
+      className={
+        cellData.isVisuallyHidden ? "subnets-table__visually-hidden" : ""
+      }
+    >
+      {cellData.href ? <a href={cellData.href}>{children}</a> : children}
+    </span>
+  </TableCellComponent>
+);
+
+export const FabricRow = memo(
+  ({ columns }: SubnetsTableRow): JSX.Element => (
+    <TableRowComponent
+      aria-label={columns.fabric.label || undefined}
+      style={{
+        transition: "opacity 250ms",
+        opacity: 1,
+      }}
```

Comment:
What does this do? Could it be a class instead?

Reply:
Sorry, this shouldn't be here (this is back from when I considered transitioning in new rows). One more reason for smaller PRs, missed it.